src/main/resources/static/index.html

```diff
@@ -15,11 +15,14 @@
 <script src="modules/auth/AuthService.js"></script>
 <script src="modules/auth/AuthCtrl.js"></script>
 
+
 <!-- Bootstrap core JavaScript
 ================================================== -->
 <!-- Placed at the end of the document so the pages load faster -->
 <script src="js/vendor/jquery/jquery.min.js"></script>
 <script src="js/vendor/bootstrap/bootstrap.min.js"></script>
```

Comment:
Bootstrap components will not work properly in angular. Check this library https://angular-ui.github.io/bootstrap/